packages/babel-helpers/src/helpers.js

```diff
@@ -413,6 +416,29 @@ helpers.interopRequireWildcard = template(`
   })
 `);
 
+helpers.specRequireInterop = template(`
+  (function (obj) {
+    if (obj && obj.__esModule) {
+      return obj;
+    } else {
+      var newObj = Object.create(null, {
+          default: {
+            value: obj,
+            writable: true,
+            enumerable: true
+          },
+          __esModule: {
```

Comment:
I was originally suggesting having two keys - keeping `__esModule`, and adding `__esSpecModule` or something. Checking enumerability to infer `__esSpecModule` is fine, although not as nicely explicit imo as a second special property.